frontend/src/interfaces/uiConfig.ts

```diff
@@ -30,6 +30,7 @@ export interface IUiConfig {
     strategySegmentsLimit?: number;
     frontendApiOrigins?: string[];
     resourceLimits: ResourceLimitsSchema;
+    oidcLocked?: boolean;
```

Comment:
Now that I think about it, maybe we should be a bit more specific about this name. It doesn't really tell us why OIDC is locked. How about something like `oidcConfiguredThroughEnv`?

Reply:
Took your suggestion. It made sense to me. Updated.